base/uk.ac.stfc.isis.ibex.opis/resources/CAENV895_single.opi

```diff
@@ -1,27 +1,27 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I think all these .opi files have been autoformatted by our git hook, and then accidentally committed - I don't think they will do any damage but in the future please be careful to make sure you only commit and push files that are part of the ticket